packages/Pagination/stories/examples/InteractivePagination.js

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import { Story } from "storybook/assets/styles/common.styles";
+import Card from "@paprika/card";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Heading from "@paprika/heading";
+import Avatar from "@paprika/avatar";
+import Pagination from "../../src/Pagination";
+
+const data1 = [
```

Comment:
since this is a story and what you want is devs focusing in reading your code example, I would suggest to move all the data into a separate file and import it from the other file something like:
```
// mocks.js

export const data1 = [...]
export const data2 = [...]

```

and then in this file:

```js
import { data1, data2 } from "./ mocks"
```

And everything should keep working correctly but without data cluttering.